test/lib/modules/apostrophe-salesforce-connect/connect.cer

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Certs must not be committed to a public module. These tests should reside in a private test project to avoid this security problem.

Reply:
Since this touched a public repo, and there are people who troll for this stuff, if this cert has any access to something in salesforce it will have to be invalidated and recreated.

Reply:
Especially since the private key is in here. There are situations where cert alone but no private key would be fine.